app-cdr/cddetect/cddetect-2.1.ebuild

```diff
@@ -1,21 +1,22 @@
-# Copyright 1999-2012 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=4
+EAPI=7
+
 inherit toolchain-funcs
 
-DESCRIPTION="A tool for detecting the type of a CD/DVD without mounting it"
+DESCRIPTION="Tool for detecting the type of a CD/DVD without mounting it"
 HOMEPAGE="http://www.bellut.net/projects.html"
 SRC_URI="http://www.bellut.net/files/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 S=${WORKDIR}
 
 src_prepare() {
+	default
 	sed -i -e '1i#include <limits.h>' ${PN}.c || die #337628
```

Comment:
please turn this into a patch and put it in `PATCHES`

Reply:
Done :)